foundry.toml

```diff
@@ -0,0 +1,15 @@
+[default]
+src = 'src'
+out = 'out'
+libs = ['lib']
+solc-version = "0.8.7"
+optimizer = true
+optimizer_runs = 1000000
+verbosity = 1
+fuzz_runs = 5 
```

Comment:
I suggest we increase `fuzz_runs`, as forge does fuzzing by [default](https://github.com/gakonst/foundry/tree/master/forge#fuzzing-go-beyond-unit-testing).

Reply:
I think that 5 is a good default value. It's used when running tests locally, so increasing it too much would slow down the developer flow. The CI is a different story, it has time to crunch a lot of runs.